.github/workflows/stable-release.yml

```diff
@@ -18,6 +16,8 @@ jobs:
     steps:
       - name: Checkout KubeArmor code
         uses: actions/checkout@v2
+        with:
+          ref: main
```

Comment:
The primary problem with this github action is that the stable tag will be updated very frequently ... i.e., on any push to the main branch or any v* branches!

In docker hub this will end up in showing wrong (for us) update date ...

Can you put a check to only do `docker image tag` if the STABLE-RELEASE version in main branch points to the current version branch?